docker-local/dfsp_a/dfsp_a_backend.env

```diff
@@ -30,7 +30,7 @@ DFSP_ID=dfspa
 MODEL_DATABASE=./model.sqlite
 
 # Outbound API endpoint (It might be a container in the compose file so remember the networking IP)
-OUTBOUND_ENDPOINT=http://dfspa-scheme-adapter:5002
+OUTBOUND_ENDPOINT=http://dfspa-sdk-scheme-adapter:5002
 
```

Comment:
don't we need here also THIRDPARTY_OUTBOUND_ENDPOINT?

Reply:
Wasn't aware that the simulator was updated for the thirdparty-scheme-adapter. Nevertheless better to add that endpoint now :+1:. done